.github/workflows/ci.yml

```diff
@@ -0,0 +1,110 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+env:
+  SERVER_ASSET: trypurescript-server
+  CLIENT_ASSET: trypurescript-client
+
+jobs:
+  build_server:
+    name: Build server
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: haskell/actions/setup@v1
+        with:
+          enable-stack: true
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.stack
+            .stack-work
+          key: ${{ runner.os }}-stack-${{ hashFiles('stack.yaml.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-stack
+
+      - name: Build server code
+        run: |
+          timeout 45m stack --no-terminal -j1 build
```

Comment:
GitHub actions’ time limits are much more generous than Travis’, so I think we can probably remove the `timeout`